service/sender.go

```diff
@@ -14,101 +14,96 @@ import (
 	"github.com/textileio/textile-go/pb"
 )
 
-// sender is a reusable peer stream
-type sender struct {
-	protocol   protocol.ID
-	stream     inet.Stream
-	writer     ggio.WriteCloser
-	reader     ggio.ReadCloser
-	mux        sync.Mutex
-	pid        peer.ID
-	service    *Service
-	singleMsg  int
-	invalid    bool
-	requests   map[int32]chan *pb.Envelope
-	requestMux sync.Mutex
+type messageSender struct {
```

Comment:
are these all short/one letter vars because we need this to be small? otherwise, kinda hard to parse as a mere human...

Reply:
i just went with exactly what's in the `go-libp2p-kad-dht` module as far as variable

Reply:
thought that might be the case. sounds good 👍